### PR TITLE
feat(tables): Get the available tables or the tables that the waitress is  attending

### PR DESCRIPTION
**Pull request by jlazo**
Get the available tables or the tables that the waitress is  attending

### CODE REVIEW

File: src/modules/table/table.service.ts [hunk 1]
```diff
@@ -23,8 +32,30 @@ export class TableService {
...
+    }
+    let tables;
+    if (user.role.name == RoleEnum.ADMIN) {
+      tables = await this._tableRepository.find();
+    } else {
+      const orders = await this._orderRepository.find({
+        waitressId: filter.userId,
+        status: Not(Equal(OrderStatus.FINISHED)),
+      });
+      const wheres: any = [{ isAvailable: true }];
```
Comment (jcmexdev) [inline src/modules/table/table.service.ts]: should we add is_active filter?

**Review by jcmexdev — COMMENTED** [review]
I´d  like to add is_active filter to table query but i'm thinking if it is necessary what do you think?

File: src/modules/table/table.service.ts (hunk 1 above)
Comment (jlazo) [reply src/modules/table/table.service.ts]: Ready

**Comment by jlazo** [comment]
> I´d like to add is_active filter to table query but i'm thinking if it is necessary what do you think?

That's correct.

**Review by jcmexdev — APPROVED** [review]
Good Job 😎👌

**Review by AlanDavd — APPROVED** [review]
LGTM

**Pull request merged**